src/dataset.ts

```diff
@@ -98,6 +100,10 @@ export interface Structure {
      * expressed in Angströms.
      */
     cell?: number[];
+    /**
+     * shapes of the particles / atoms in the viewer
+     */
+    shape?: [{ [key: string]: any }];
```

Comment:
This should not matter for the rendering, but something like this would be nicer for typing:

```suggestion
    shape?: [{ [key: string]: SphereData | EllipsoidData }];
```

with the definition of `SphereData`/`EllipsoidData` matching what's checked below